.env

```diff
@@ -6,4 +6,5 @@ OAUTH_CLIENT_SECRET=askTeammate
 IPSTACK_KEY=askTeammate
 NEXT_PUBLIC_DS_GLOBAL_HEADER=https://ds-header.nypl.org/header.min.js?containerId=nypl-header
 NEXT_PUBLIC_DS_GLOBAL_FOOTER=https://ds-header.nypl.org/footer.min.js?containerId=nypl-footer
-NEXT_PUBLIC_ADOBE_ANALYTICS_TAG=https://assets.adobedtm.com/1a9376472d37/8519dfce636d/launch-672b7e7f98ee.min.js
\ No newline at end of file
+NEXT_PUBLIC_ADOBE_ANALYTICS_TAG=https://assets.adobedtm.com/1a9376472d37/8519dfce636d/launch-672b7e7f98ee.min.js
+COOKIE_DOMAIN=.nypl.org
```

Comment:
Just a reminder that the task definition will need this on QA and production. DevOps will need to update the base CF or TR template to include this new env var.

Reply:
i just filled out a DOPS ticket for this update, with a target date of this coming tuesday.

Reply:
Great, thanks